### ISSUE TITLE
xetex build tests fail due to fontspec v2.6 defining ``\strong`` 

### ISSUE THREAD

**Issue by jfbu** [sphinx-doc/sphinx#3428]
latest update to [fontspec](http://www.ctan.org/pkg/fontspec) defines ``\strong``. This causes build failure for PDF output when ``latex_engine= 'xelatex'`` 

```latex
! LaTeX Error: Command \strong already defined.
               Or name \end... illegal, see p.192 of the manual.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
```

fortunately, Sphinx anticipated this some time ago ;-)

```latex
** (sphinx) defining (legacy) text style macros without \sphinx prefix
** if clashes with packages, set latex_keep_old_macro_names=False in conf.py

```
Hence, default configuration for ``xelatex`` should do this  ``latex_keep_old_macro_names=False``. The problem will also arise with ``lualatex`` if it uses ``fontspec``. Currently default setting of Sphinx with ``lualatex`` does not yet do it because it needed to pass tests with Ubuntu Precise which has TeXLive 2009 whose ``fontspec`` did not yet have good support for ``lualatex`` (see https://github.com/sphinx-doc/sphinx/issues/3070#issuecomment-255940439)

### Problem
- LaTeX PDF build fails

#### Procedure to reproduce the problem
- set ``latex_engine = 'xelatex'`` in conf.py and run (with Sphinx default font configuration for xelatex) with fully updated TeXLive 2016 install.

#### Error logs / results
as above

#### Expected results
latexpdf compiles

### Reproducible project / your project
any project using xelatex

### Environment info
- fully updated TeXLive 2016 (February 17, 2016)
- Sphinx version: 1.5.2

**Comment by jfbu** [comment]
fontspec link: https://github.com/wspr/fontspec/commit/f3f4132305f75e5913f5ac893083d5d43c690e46

**Comment by jfbu** [comment]
the immediate problem is fixed at b0a5ec1 for 1.5.3 release. Discussion on future policy continues at #3429.
